horreum-web/src/About.tsx

```diff
@@ -92,11 +94,6 @@ export default function About() {
                 title="About Horreum"
                 isOpen={isModalOpen}
                 onClose={() => setModalOpen(false)}
-                actions={[
-                    <Button key={0} onClick={() => setModalOpen(false)}>
-                        Close
-                    </Button>,
-                ]}
```

Comment:
Also removed the "Close" button as there is already the default right-most "X" one